src/extension/generalMobilePlatform.ts

```diff
@@ -81,6 +82,35 @@ export class GeneralMobilePlatform {
         return Promise.resolve();
     }
 
+    public modifyOptFromRunArgs(optName: string, value: any): void {
+        const isBinary = isBoolean(value);
+        const optIdx = this.runArguments.indexOf(optName);
+        if (optIdx > -1) {
+            if (isBinary && !value) {
+                this.runArguments.splice(optIdx, 1);
+            }
+            if (!isBinary) {
+                if (value === null && value === undefined) {
```

Comment:
According to strict equality in JS `null` is not equal to `undefined`. So, this condition cannot be `true`.